app/controllers/crime_applications_controller.rb

```diff
@@ -5,6 +5,7 @@ class CrimeApplicationsController < ApplicationController
   def index
     # TODO: scope will change as we know more
     @applications = CrimeApplication
+                    .viewable
```

Comment:
Hmm, I think this might not be necessary, or early optimisation. We are already scoping applications by having an applicant record and in addition, this record having at least first_name entered.

Can't think of a scenario (apart from faking it through the console) where we would return applications in an `initialised` status.

I think if we have to do all this, then I'm more inclined to remove the initialised status as it is clearly adding extra complexity and mental overhead.

Reply:
Yeah I think you're right on balance - i'll remove for now